Firestore/src/Aggregate.php

```diff
@@ -68,6 +78,48 @@ public static function count()
         return $count;
     }
 
+    /**
+     * Creates sum aggregation properties.
+     *
+     * Example:
+     * ```
+     * $sum = Aggregate::sum('field_to_aggregate_upon');
+     * ```
+     * @param string $field The relative path of the field to aggregate upon.
+     * @return Aggregate
+     */
+    public static function sum($field)
+    {
+        $sum = new Aggregate(self::TYPE_SUM);
+        $sum->props[$sum->aggregationType] = [
+            'field' => [
+                'fieldPath' => $field
+            ]
+        ];
+        return $sum;
+    }
+
+    /**
+     * Creates average aggregation properties.
+     *
+     * Example:
+     * ```
+     * $avg = Aggregate::avg('field_to_aggregate_upon');
+     * ```
+     * @param string $field The relative path of the field to aggregate upon.
+     * @return Aggregate
+     */
+    public static function avg($field)
+    {
+        $avg = new Aggregate(self::TYPE_AVG);
+        $avg->props[$avg->aggregationType] = [
+            'field' => [
+                'fieldPath' => $field
+            ]
+        ];
```

Comment:
Do you think a private function in `Aggregate` class to setField is better? Since there's a duplication of code here.

Reply:
Thanks, nice callout. I figured we can extract the whole aggregate creation process and thus did that. Now it looks a lot better.